lib/rubocop/cop/lint/circular_argument_reference.rb

```diff
@@ -62,11 +62,6 @@ def on_optarg(node)
 
         def check_for_circular_argument_references(arg_name, arg_value)
           case arg_value.type
-          when :send
```

Comment:
IMO this shouldn't be a `case` anymore. :-) The subsequent comment also becomes redundant.

Reply:
Yes, indeed. I updated and rebased.

Reply:
And I can totally suggest creating a cop for an useless `case`. :-)

Reply:
I see. A cop suggesting `if` (or `unless`) for `case` who has only one `when` seems to be useful 😃

Reply:
Indeed. I think we have the opposite cop (suggesting to use `case` instead of some `if/elsif/else`s, so this would be a nice complement.